Installation of rust-999 fails

`sudo emerge -a dev-lang/rust:nightly` seems to fail with:

```
configure: error: Option '--disable-rustbuild' is not recognized
```

Removing that option from the ebuild seems to fix the issue. It seems this option has been removed from the configure script [19 days ago](https://github.com/rust-lang/rust/pull/39774), and it has been useless for a bit longer than that.

--- a/dev-lang/rust/rust-999.ebuild
+++ b/dev-lang/rust/rust-999.ebuild
@@ -71,7 +71,6 @@ src_configure() {
 		--release-channel=${release_channel%%/*} \
 		--extra-filename=${postfix} \
 		--disable-manage-submodules \
-		--disable-rustbuild \
 		--default-linker=$(tc-getBUILD_CC) \
 		--default-ar=$(tc-getBUILD_AR) \
 		--python=${EPYTHON} \
@@ -90,7 +89,7 @@ src_configure() {
 }
 
 src_compile() {
-	emake VERBOSE=1
+	emake dist VERBOSE=1
 }
 
 src_install() {

